packages/lib/gpcircuits/src/index.ts

```diff
@@ -7,6 +7,7 @@ export * from "./multituple";
 export * from "./numeric-value";
 export * from "./object";
 export * from "./owner";
```

Comment:
As mentioned elsewhere, I suggest putting both modules in the same file, or else calling this one ownerV3